editor.planx.uk/src/components/Feedback/FeedbackForm.tsx

```diff
@@ -52,8 +54,9 @@ const FeedbackForm: React.FC<FormProps> = ({ inputs, handleSubmit }) => {
   return (
     <Formik initialValues={initialValues} onSubmit={handleSubmit}>
       <StyledForm>
+        <FeedbackDataDisclaimer />
```

Comment:
Split this `<FeedbackDisclaimer>` into to imports for each header, one focused on personal / financial data as `<FeedbackDataDisclaimer>` and one focused on the monitoring of the feedback `<FeedbackMonitoringDisclaimer>`.

Situated them in the same file as they both serve similar functions and contextually relevant, but seperate enough in structure to have as separate components. 

Be good to have thoughts on this pattern...

Reply:
@DafyddLlyr  thanks! Switched it to this. It felt interesting cause the text was contextually linked but had a different structure, with a link in the text. Just passed it now as it is. Hope this is cleaner